src/schema/graphql-schema.ts

```diff
@@ -2,9 +2,12 @@ import { ApolloServer } from '@apollo/server';
 import { gql } from 'graphql-tag';
 import { PrismaClient } from '@prisma/client';
 import { DateTimeResolver } from 'graphql-scalars';
+import bcrypt from 'bcrypt';
```

Comment:
Poderia utilizar scrypt que já vem no node: https://nodejs.org/api/crypto.html#cryptoscryptsyncpassword-salt-keylen-options

ai não existe a necessidade de adicionar outra biblioteca.

Existe uma discussão do que é melhor, scrypt ou bcrypt: https://security.stackexchange.com/questions/26245/is-bcrypt-better-than-scrypt
o bcrypt é mais antigo e até hoje não foi quebrado, o scrypt é mais novo (o que do ponto de vista de segurança não é melhor). A muinha conclusão é que tanto faz, os dois implementados direito são seguros o bastante, mas o scrypt tem a vantagem de já estar no node